File: dataset/java2python/java/C_PROGRAM_FIND_SECOND_FREQUENT_CHARACTER.java
char f_gold ( String str ) {
  int [ ] count = new int [ NO_OF_CHARS ] ;
  int i ;
  for ( i = 0 ;
  i < str . length ( ) ;
  i ++ ) ( count [ str . charAt ( i ) ] ) ++ ;
  int first = 0 , second = 0 ;
  for ( i = 0 ;
  i < NO_OF_CHARS ;
  i ++ ) {
    if ( count [ i ] > count [ first ] ) {
      second = first ;
      first = i ;
    }
    else if ( count [ i ] > count [ second ] && count [ i ] != count [ first ] ) second = i ;
  }
  return ( char ) second ;
}

File: result/repair/transcoder/java2python/python_repair/C_PROGRAM_FIND_SECOND_FREQUENT_CHARACTER.py
def f_gold ( str ) :
    count = [ 0 ] * NO_OF_CHARS
    i = 0
    for c in str :
        ( count [ c ] , count [ c ] ) = count [ c ] , count [ c ]
    first , second = 0 , 0
    for c in NO_OF_CHARS :
        if (count[i]>count[first]) :
            second , first = first , c
        elif (count[i]>count[second] and count[i]!=count[first]) :
            second , first = c , first
    return chr ( second ) , first , second
